mpas_analysis/ocean/time_series_ohc.py

```diff
@@ -292,48 +318,271 @@ def run(self):  # {{{
         for regionIndex in regionIndicesToPlot:
             region = regions[regionIndex]
 
-            ohc = dsOHC.ohc.isel(nOceanRegionsTmp=regionIndex)
+            # Plot temperature, salinity and OHC anomalies (and full fields)
+            # trends with depth
+
+            #  First T vs depth/time
+            x = ds.Time.values
+            y = depth
+            z = ds.avgLayerTemperatureAnomaly.isel(nOceanRegionsTmp=regionIndex)
+            z = z.transpose()
+
+            colorbarLabel = '[$^\circ$ C]'
+            xLabel = 'Time [years]'
+            yLabel = 'Depth [m]'
+
+            title = 'Temperature Anomaly, {} \n {}'.format(plotTitles[regionIndex], mainRunName)
+
+            figureName = '{}/{}.png'.format(self.plotsDirectory, self.filePrefixes['TAnomalyZ', region])
+
+            (colormapName, colorbarLevels) = setup_colormap(config,
+                                                            configSectionName,
+                                                            suffix='TemperatureAnomaly')
+
+            contourLevels = \
+                config.getExpression(configSectionName,
+                                     'contourLevels{}'.format('TemperatureAnomaly'),
+                                     usenumpyfunc=True)
+   
+            plot_vertical_section(config, x, y, z,
+                                  colormapName, colorbarLevels, contourLevels,
+                                  colorbarLabel, title, xLabel, yLabel,
+                                  figureName, linewidths=1, xArrayIsTime=True,
+                                  N=movingAveragePointsHovmoller, calendar=calendar)
+
+            caption = 'Trend of {} Temperature Anomaly vs depth from Year 0001'.format(region)
```

Comment:
One rather minor thing I just noticed is that `0001` shouldn't be hard coded here.  Instead, this should be `'... Year {:04d}'.format(region, timeStartFirstYear)`.